### PR TITLE
Update export names

### PR DESCRIPTION
**Pull request by zeppelin**
* Merged master into canary
* Use the module names proposed by the core team

### CODE REVIEW

File: app-shims.js [hunk 1]
```diff
@@ -3,49 +3,231 @@
...
+        'default': Ember.TextArea
+      },
+      'ember-components/text-field': {
+        'default': Ember.TextField
+      },
+      'ember-controller': {
         'default': Ember.Controller
       },
-      'ember/controllers/array': {
+      'ember-controller/array': {
```
Comment (stefanpenner) [inline app-shims.js]: should we just drop this one, as this is going away?

File: app-shims.js [hunk 2]
```diff
@@ -3,49 +3,231 @@
...
-      'ember/controllers/object': {
+      'ember-controller/inject': {
+        'default': Ember.inject.controller
+      },
+      'ember-controller/object': {
         'default': Ember.ObjectController
       },
-      'ember/component': {
-        'default': Ember.Component
+      'ember-controller/proxy': {
```
Comment (stefanpenner) [inline app-shims.js]: should we just drop this one, as this is going away?

File: app-shims.js [hunk 3]
```diff
@@ -3,49 +3,231 @@
...
+        'default': Ember.ArrayProxy
       },
-      'ember/view': {
-        'default': Ember.View
+      'ember-controllers/sortable': {
```
Comment (stefanpenner) [inline app-shims.js]: should we just drop this one, as this is going away in 2.0?

File: app-shims.js [hunk 4]
```diff
@@ -3,49 +3,231 @@
...
+      },
+      'ember-debug/data-adapter': {
+        'default': Ember.DataAdapter
+      },
+      'ember-deprecations': {
+        'deprecate':      Ember.deprecate,
+        'deprecateFunc':  Ember.deprecateFunc
+      },
+      'ember-enumerable': {
+        'default': Ember.Enumerable
```
Comment (stefanpenner) [inline app-shims.js]: should we just drop this one, as this is *hopefully* is going away in 2.0?

**Comment by stefanpenner** [comment]
@wycats @tomdale, its kinda crappy to do

```js
import { get } from 'ember-metal/get'
import { set } from 'ember-metal/set';
```

this pair really gets used often together, the following may be nicer

```
import { get, set } from 'ember-metal/properties'
```

File: app-shims.js [hunk 5]
```diff
@@ -3,49 +3,231 @@
...
+        'sync':         Ember.run.sync,
+        'throttle':     Ember.run.throttle
+      },
+      'ember-service': {
+        'default': Ember.Service
+      },
+      'ember-service/inject': {
+        'default': Ember.inject.service
+      },
+      'ember-set/ordered': {
```
Comment (stefanpenner) [inline app-shims.js]: @tomdale / @wycats do we/es6 have an un-ordered set?

File: app-shims.js [hunk 6]
```diff
@@ -3,49 +3,231 @@
...
+      'ember-components/select': {
+        'default': Ember.Select
+      },
+      'ember-components/text-area': {
+        'default': Ember.TextArea
+      },
+      'ember-components/text-field': {
+        'default': Ember.TextField
+      },
+      'ember-controller': {
```
Comment (stefanpenner) [inline app-shims.js]: should we just drop this one, as this is going away in 2.0?

File: app-shims.js (hunk 1 above)
Comment (rwjblue) [inline app-shims.js]: we will still have these modules in an addon so it makes some sense to expose IMHO

File: app-shims.js (hunk 2 above)
Comment (rwjblue) [inline app-shims.js]: we will still have these modules in an addon so it makes some sense to expose IMHO

File: app-shims.js (hunk 3 above)
Comment (rwjblue) [inline app-shims.js]: It was listed in our listing (discussed at last Face to Face) to move to an addon, which is why it is included here.

File: app-shims.js (hunk 4 above)
Comment (rwjblue) [inline app-shims.js]: Yes, lets drop for now.

Comment (stefanpenner) [inline app-shims.js]: is this the same as the above, included via addon?

File: app-shims.js (hunk 3 above)
Comment (stefanpenner) [inline app-shims.js]: :+1:

File: app-shims.js (hunk 2 above)
Comment (stefanpenner) [inline app-shims.js]: :+1:

File: app-shims.js (hunk 1 above)
Comment (stefanpenner) [inline app-shims.js]: :

File: app-shims.js [hunk 7]
```diff
@@ -3,49 +3,231 @@
...
+      },
+      'ember-location/none': {
+        'default': Ember.NoneLocation
+      },
+      'ember-map': {
+        'default':      Ember.Map,
+        'withDefault':  Ember.MapWithDefault
+      },
+      'ember-metal/computed': {
+        'default': Ember.computed
```
Comment (stefanpenner) [inline app-shims.js]: seems like we might want to export the rest of the computed macros? `alias` `readOnly` `gte` lt` etc

Comment (rwjblue) [inline app-shims.js]: @stefanpenner - They are done in a loop below this section.

File: app-shims.js (hunk 4 above)
Comment (rwjblue) [inline app-shims.js]: @stefanpenner - Yeah, for 2.0 it will be an addon.

File: app-shims.js [hunk 8]
```diff
@@ -3,49 +3,231 @@
...
+        'cancel':       Ember.run.cancel,
+        'debounce':     Ember.run.debounce,
+        'end':          Ember.run.end,
+        'join':         Ember.run.join,
+        'later':        Ember.run.later,
+        'next':         Ember.run.next,
+        'once':         Ember.run.once,
+        'schedule':     Ember.run.schedule,
+        'scheduleOnce': Ember.run.scheduleOnce,
+        'sync':         Ember.run.sync,
```
Comment (stefanpenner) [inline app-shims.js]: can we kill this one?

File: app-shims.js (hunk 7 above)
Comment (stefanpenner) [inline app-shims.js]: :+1:

File: app-shims.js [hunk 9]
```diff
@@ -3,49 +3,231 @@
...
+      'ember-view/container': {
+        'default': Ember.ContainerView
       }
     };
 
     // populate `ember/computed` named exports
+    shims['ember-computed'] = {};
     for (var key in Ember.computed) {
-      shims['ember/computed'][key] = Ember.computed[key];
+      shims['ember-computed'][key] = Ember.computed[key];
```
Comment (stefanpenner) [inline app-shims.js]: ah i see

Comment (stefanpenner) [inline app-shims.js]: those also exposes peoples own or add-on included macros residing on`Ember.computed`, we should likely just make an array with the ones we explicitly want to expose as api.

File: app-shims.js (hunk 5 above)
Comment (mmun) [inline app-shims.js]: Calling Set ordered is confusing. An "ordered set" typically refers to a set of objects together with an order relation. This should just be called set. Technically the implementation is a "linked hash set".

File: app-shims.js [hunk 10]
```diff
@@ -3,49 +3,231 @@
...
+      'ember-test': {
+        'default': Ember.Test
+      },
+      'ember-test/adapter': {
+        'default': Ember.Test.Adapter
+      },
+      'ember-test/qunit-adapter': {
+        'default': Ember.Test.QUnitAdapter
+      },
+      'ember-utils': {
```
Comment (mmun) [inline app-shims.js]: Is there a better name than `utils`? We can split it up.

File: app-shims.js [hunk 11]
```diff
@@ -3,49 +3,231 @@
...
+        'isEmpty':    Ember.isEmpty,
+        'isNone':     Ember.isNone,
+        'isPresent':  Ember.isPresent,
+        'tryInvoke':  Ember.tryInvoke,
+        'typeOf':     Ember.typeOf
+      },
+      'ember-view': {
+        'default': Ember.View
+      },
+      'ember-view/collection': {
```
Comment (mmun) [inline app-shims.js]: Should be `ember-views/collection` to match with `ember-components`. May as well be `ember-views/collection-view`.

File: app-shims.js (hunk 9 above)
Comment (stefanpenner) [inline app-shims.js]: @rwjblue this one is the only one that should be handled before merge.

Comment (zeppelin) [inline app-shims.js]: @stefanpenner I'll update to whatever you guys decide this should be.

File: app-shims.js [hunk 12]
```diff
@@ -3,49 +3,222 @@
 
   function processEmberShims() {
     var shims = {
+      // TODO:
+      // - Ember.Copyable
+      // - Ember.on
```
Comment (zeppelin) [inline app-shims.js]: @rwjblue This also should be addressed, as the spreadsheet didn't include `Ember.on`. What would the module name be?

**Comment by stefanpenner** [comment]
Yay

**Comment by teddyzeenny** [comment]
Nice!!

**Pull request merged**